planner/core/errors.go

```diff
@@ -147,6 +146,12 @@ var (
 
 func init() {
 	mysqlErrCodeMap := map[terror.ErrCode]uint16{
+		codeUnsupportedType:              mysql.ErrUnsupportedType,
```

Comment:
Please add some tests for this map.

Reply:
Could you give some hint on how to add tests for this map?

Reply:
maybe like https://github.com/pingcap/tidb/pull/13300/files#diff-8715d6cdb7a45a830b51e3bb85466eeaR26